eos-tech-support/eos-usb-internet-writer

```diff
@@ -0,0 +1,229 @@
+#!/bin/bash -e
+set -o pipefail
+
+eos_write_live_image=$(dirname "$0")/eos-write-live-image
+if [ ! -f "$eos_write_live_image" ]; then
+    eos_write_live_image='eos-write-live-image'
+fi
+
+args=$(getopt -o t:h \
+    -l tmp-dir: \
+    -l debug \
+    -l help \
+    -n "$0" -- "$@")
+eval set -- "$args"
+
+tmp_dir=/tmp/eos-usb-internet-writer-tmp
+debug=
+
+usage() {
+    local self
+    self=$(basename "$0")
+    cat <<EOF
+Usage:
+    $self [options] -- IMAGE_OPTIONS
+
+Arguments:
+    IMAGE_OPTIONS       Everything after -- is passed to the $eos_write_live_image
+                        command.
+
+Options:
+    -t, --tmp-dir PATH  Temporary directory to use for downloading files and
+                        save the USB key files to copy, user is in charge of
+                        cleaning this directory if required, the command will
+                        try to all the content from this directory.
+                        (default: $tmp_dir)
+        --debug         Enable debugging output
+    -h, --help          Show this message
+EOF
```

Comment:
I realize it's possible to pass it after --, but do you think it would be okay to add an eos_windows_launcher option here? Then this script would deal with the --windows-tool and --label options for eos-write-live-image on its own, like we do in the write-usb command for Hack. That way it could be a required argument. I think that makes sense since it's a key part of the eos-usb-internet thing, and then the options we add after -- would be strictly optional things related to the Endless OS image.

Reply:
All right, maybe as suggested here https://github.com/endlessm/eos-meta/pull/632#discussion_r468662709 it's better to create a higher level set of flags for this command and let this command figure out most of the flags for `eos-write-live-image`.